modules/features/manatime_feature_timesheets/manatime_feature_timesheets.module

```diff
@@ -380,6 +413,45 @@ function manatime_feature_timesheets_add_row_callback($node, $ajax) {
 }
 
 /**
+ * Ajax menu callback - Add row.
+ */
+function manatime_feature_timesheets_add_time_entry_callback($node, $project_id, $task_id, $number_day, $ajax) {
+  if ($ajax) {
+    ctools_include('ajax');
+    ctools_include('modal');
+
+    $form_state = array(
+      'ajax' => TRUE,
+      'title' => t('Add time entry'),
+    );
+
+    $form_state['build_info']['args'] = array(
+      $node,
+      $project_id,
+      $task_id,
+      $number_day,
+    );
+
+    // Use ctools to generate ajax instructions for the browser to create
+    // a form in a modal popup.
+    $output = ctools_modal_form_wrapper('manatime_feature_timesheets_add_time_entry_form', $form_state);
+
+    // If the form has been submitted, there may be additional instructions
+    // such as dismissing the modal popup.
+    if (!empty($form_state['ajax_commands'])) {
+      $output = $form_state['ajax_commands'];
+    }
+
+    // Return the ajax instructions to the browser via ajax_render().
+    print ajax_render($output);
+    drupal_exit();
+  }
+  else {
+    return drupal_get_form('manatime_feature_timesheets_add_time_entry_form');
```

Comment:
if you're manually invoking drupal_get_form, you SHOULD pass the parameters somehow (after form name) so that we really support non-js enabled browsers